src/docfx/serve/lsp/credentialRefresh/CredentialRefreshParams.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using MediatR;
+using OmniSharp.Extensions.JsonRpc;
+
+namespace Microsoft.Docs.Build
+{
+    [Parallel]
```

Comment:
Should this be `serial`? How would client process multiple sign in request at the same time
```suggestion
    [Serial]
```